### PR TITLE
Added the ability to configure the max message size for the client to…

### PR DESCRIPTION
**Pull request by sirgantrithon**
… receive

### CODE REVIEW

File: src/main/java/com/hubrick/vertx/kafka/consumer/KafkaConsumerVerticle.java [hunk 1]
```diff
@@ -55,7 +55,8 @@ public void start() throws Exception {
                 config.getInteger(KafkaConsumerProperties.KEY_MAX_RETRIES, Integer.MAX_VALUE),
                 config.getInteger(KafkaConsumerProperties.KEY_INITIAL_RETRY_DELAY_SECONDS, 1),
                 config.getInteger(KafkaConsumerProperties.KEY_MAX_RETRY_DELAY_SECONDS, 10),
-                config.getLong(KafkaConsumerProperties.EVENT_BUS_SEND_TIMEOUT, DeliveryOptions.DEFAULT_TIMEOUT)
+                config.getLong(KafkaConsumerProperties.EVENT_BUS_SEND_TIMEOUT, DeliveryOptions.DEFAULT_TIMEOUT),
+                config.getLong(KafkaConsumerProperties.CLIENT_MESSAGE_MAX_BYTES, 1048576L)
```
Comment (marcust) [inline src/main/java/com/hubrick/vertx/kafka/consumer/KafkaConsumerVerticle.java]: Where does this magic number come from? Can you add a constant and comment that?

**Comment by eemmiirr** [comment]
Could you just please add this to the README

**Comment by marcust** [comment]
@sirgantrithon can you please rebase this on master and add the configuration option to the documentation?

**Comment by marcust** [comment]
Closed due to inactivity

**Pull request closed without merging**